CRITICAL - 2020-06-29 00:01:56 --> Use of undefined constant APP_NAME - assumed 'APP_NAME' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\kusumasyariah\app\Views\auth\login.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 4, Array)
#1 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\View\View.php(236): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Common.php(1071): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\xampp\htdocs\kusumasyariah\app\Controllers\Auth.php(57): view('auth/login', Array)
#4 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Auth->login()
#5 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\kusumasyariah\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-06-29 00:08:14 --> Use of undefined constant APP_NAME - assumed 'APP_NAME' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\kusumasyariah\app\Views\auth\login.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 4, Array)
#1 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\View\View.php(236): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Common.php(1071): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\xampp\htdocs\kusumasyariah\app\Controllers\Auth.php(57): view('auth/login', Array)
#4 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Auth->login()
#5 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\kusumasyariah\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-06-29 01:04:24 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Model.php(482): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\kusumasyariah\app\Controllers\Auth.php(28): CodeIgniter\Model->first()
#5 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Auth->login()
#6 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\kusumasyariah\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2020-06-29 01:04:24 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\kusumasyariah\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\kusumasyariah\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Auth), true)
#2 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Debug\Exceptions.php(298): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\kusumasyariah\vendor\codeigniter4\framework\system\Debug\Exceptions.php(171): CodeIgniter\Debug\Exceptions->render(Object(mysqli_sql_exception), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
